### ISSUE TITLE
Set "target" when adding new post in "StatusUpdateForm"

### ISSUE THREAD

**Issue by TamaWhiu** [GetStream/react-activity-feed#3]
Please suggest an approach to set "target" when adding a new post in "StatusUpdateForm" - currently searching thru code/docs.

eg. [https://getstream.io/docs/js/#adding-activities](https://getstream.io/docs/js/#adding-activities) 
`// Add an activity to the feed`
`var activity = {actor: 'User:1', verb: 'post', object: 'Place:42', target: 'Group:1'}; `

**Comment by JelteF** [comment]
I just released v0.5.1 of this library with support for this in the form of the `modifyActivityData` prop:
https://getstream.github.io/react-activity-feed/#!/StatusUpdateForm

You can use it like this:
```jsx
<StatusUpdateForm
  modifyActivityData={(data) => ({...data, target: 'Group:1'})}
/>
```

Let me know if it doesn't work for you.

**Comment by TamaWhiu** [comment]
Great - works a treat, also successfully tried out adding "to:"

   ```javascript
    <StatusUpdateForm
      modifyActivityData={
                (data) => ({
                  ...data,  
                  target: this.props.groupId,
                  to:[`user:${this.props.user.Id)}`]
                })
              }
      />
